BinaryEncoding.md

```diff
@@ -500,6 +504,47 @@ where a `local_name` is encoded as:
 | index | `varuint32` | the index of the function whose locals are being named |
 | local_map | `name_map` | assignment of names to local indices |
 
+#### Label names
+
+The label names subsection assigns `name_map`s to a subset of functions in the
+[function index space](Modules.md#function-index-space). This may include both
+module-defined or imported functions, but is only meaningful for module-defined
+functions. The `name_map` for a given function assigns names to labels within
+the function, with labels identified by the offset from the start of the function's
```

Comment:
I considered an implicit numbering of the control flow operators first, but the benefit to using a byte offset is that it doesn't require tracking additional state when decoding the instructions to compute some implicit label index for the instruction being decoded.